### PR TITLE
Modify change area notebook to use util functions

### PR DESCRIPTION
**Pull request by hannah-rae**
Modifying the change area estimation notebook to use the same(ish) util functions created for the (single year) area estimation notebook

This requires the following steps:
- [ ] Adapt existing notebook to use util functions
- [ ] Add code for computing area estimates from confusion matrix (was not in notebook previously)
- [ ] Test by comparing change area estimates for Tigray computed using this notebook and prior analysis (outside repo)
- [ ] Test by computing change area estimates for new ROI (e.g., Bure or Jimma)

### CODE REVIEW

**Comment by ivanzvonkov** [comment]
Is this ready for review @hannah-rae? First time I see these draft PRs 🤔

**Comment by ivanzvonkov** [comment]
@hannah-rae should this be closed or on the path to merging?
FYI on colab `pip install shapely==1.8.5` is required to avoid an cartopy import error

File: notebooks/change_area_estimation.ipynb [hunk 1]
```diff
@@ -21,1996 +21,771 @@
     "- Code for sub-regional estimates (subsetting the reference sample according to admin2 bounds, e.g.), probably as a separate notebook"
```
Comment (adebowaledaniel) [inline notebooks/change_area_estimation.ipynb:21]: <pre class="ql-syntax" spellcheck="false">Line #4.    !pip install cartopy==0.19.0.post1 -q
</pre><p>!pip install cartopy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by ivanzvonkov** [comment]
@adebowaledaniel Could you clarify what's happened here?
1. Hannah opened a PR to use some of the new util functions
2. You updated some of the util functions
Is this correct?

**Comment by adebowaledaniel** [comment]
Yes, @ivanzvonkov, It updated the following:
- the `load_ne` function to dissolve sub-region (Admin level 2) geometries into one,
- m^2 to Ha conversion,
- and the unittest for area_util.

Some tasks still need to be completed for this PR; I plan to review them in the coming week.
> * [ ]  Add code for computing area estimates from confusion matrix (was not in notebook previously)
> * [ ]  Test by comparing change area estimates for Tigray computed using this notebook and prior analysis (outside repo)
> * [ ]  Test by computing change area estimates for new ROI (e.g., Bure or Jimma)

Since the recent changes are needed, especially for single-year area estimation, I will implement them in #212 and merge them.

**Comment by bhyeh** [comment]
Close w/ merge of #314

**Pull request closed without merging**